section/session-flow.html

```diff
@@ -312,16 +306,39 @@ <h3>Payment streaming</h3>
     </div>
 
     <div class="algorithm">
-      <p>To <dfn>compute payment amount</dfn>, given |session:Session|, run these steps:</p>
+      <p>To <dfn>compute payment amount and delay</dfn>, run these steps:</p>
       <ol>
-        <li>TODO</li>
-      </ol>
-    </div>
-
-    <div class="algorithm">
-      <p>To <dfn>compute payment delay</dfn>, run these steps:</p>
-      <ol>
-        <li>TODO</li>
+        <li>Let |storage:Storage| be the result of [=get storage=].</li>
+        <li>
+          Let |maxRateOfPayPerMonth:long long| be the result of parsing
+          |storage|.{{Storage/maxRateOfPay}} as a base-10 non-negative integer.
+        </li>
+        <li>
+          If |maxRateOfPayPerMonth| is 0, then return the tuple (0, 1000).
+          <p class="note">A monthly budget of 0 disables payment attempts.</p>
+        </li>
+        <li>Let |secondsInMonth:double| be <code>30 * 24 * 60 * 60</code>.</li>
+        <li class="note">A "month" is treated as a fixed 30-day period for payment pacing calculations.</li>
+        <li>Let |defaultPaymentAmountPerSecond:double| be <code>|maxRateOfPayPerMonth| / |secondsInMonth|</code>.</li>
```

Comment:
The extension has an hourly rate of pay. The monthly rate (not same as monthly budget) might be less useful, under the assumption that `30*24*60*60` is not the time user may spend in browser (why not something like 8h a day, not 24h a day?). So, we went with a lower resolution (hourly rate). It does mean user has to configure both these values. The extension by default uses $5 for monthly budget, and $0.60 as hourly rate. https://github.com/interledger/web-monetization-budget-suggestions includes recommended budgets for various currencies.